jdl/converters/types.ts

```diff
@@ -30,3 +30,12 @@ export type JSONEntity = {
   fields?: JSONField[];
   relationships?: JSONRelationship[];
 } & Record<string, any>;
+
+export type JSONGeneratorJhipsterContent = {
+  applicationType: string;
```

Comment:
`applicationType` is not require. The only required field should be `baseName`.
```suggestion
  baseName: string;
```

Reply:
Sure, sry. I'll also `applicationType?` :-p